edgelet/edgelet-docker/src/migc_persistence.rs

```diff
@@ -8,7 +8,7 @@ use edgelet_settings::base::image::MIGCSettings;
 
 use crate::{DockerModule, Error};
 
-const TEMP_FILE: &str = "/tmp/images.txt";
+const TEMP_FILE: &str = "/tmp/images";
```

Comment:
Should we be writing to file in /tmp or should it be written to somewhere in /var/run? Guessing the latter as multiple people can write to /tmp.